temp-legacy-code/src/main/java/com/ivy/legacy/legacy/ui/theme/modal/edit/AmountModal.kt

```diff
@@ -118,6 +121,25 @@ fun BoxWithConstraintsScope.AmountModal(
                     e.printStackTrace()
                 }
             }
+        },
+        SecondaryActions = {
+            if (showPlusMinus) {
+                KeypadCircleButton(
+                    text = "+/-",
+                    testTag = "plus_minus",
+                    fontSize = 22.sp,
+                    btnSize = 52.dp,
+                    onClick = {
+                        amount = if (amount.isEmpty()) {
```

Comment:
"-0" doesn't make sense from UI/UX perspective - let's do nothing for 0. Also, this logic can be re-written in a simpler way like:
```kotlin
when {
  amount.firstOrNull() == '-' -> {
     // invert negative amount to positive
     amount = amount.dropFirst(1)
   }
   amount.isNotEmpty() -> {
     // make a positive amount -> negative
     amount = "-$amount"
   }
}
```

Sth like. If else is also good just:
- do nothing for 0
- use `dropFirst` instead of substring cuz it's more readable